pyproject.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 python = "^3.10"
 pydantic = "^2.7"
 sympy = "^1.12"
-qref = "0.9.0"
+qref = "0.10.0"
```

Comment:
Why not `^0.10.0`?

Reply:
I followed the convention we used previously. However, I agree that caret specifier would be better, so I changed it in https://github.com/PsiQ/bartiq/pull/169/commits/d675694c323a574aebe191782338a12fa8208c92